File: token-client-core/src/main/java/no/nav/security/token/support/client/core/oauth2/ClientCredentialsGrantRequest.java
package no.nav.security.token.support.client.core.oauth2;

import no.nav.security.token.support.client.core.ClientProperties;
import no.nav.security.token.support.client.core.OAuth2GrantType;

public class ClientCredentialsGrantRequest extends AbstractOAuth2GrantRequest {

    public ClientCredentialsGrantRequest(ClientProperties clientProperties) {
        super(OAuth2GrantType.CLIENT_CREDENTIALS, clientProperties);
    }
}

File: token-client-core/src/main/kotlin/no/nav/security/token/support/client/core/oauth2/ClientCredentialsGrantRequest.kt
package no.nav.security.token.support.client.core.oauth2

import com.nimbusds.oauth2.sdk.GrantType.CLIENT_CREDENTIALS
import no.nav.security.token.support.client.core.ClientProperties

class ClientCredentialsGrantRequest(clientProperties : ClientProperties) : AbstractOAuth2GrantRequest(CLIENT_CREDENTIALS, clientProperties)